### DISCUSSION TITLE
JDBC-server compatibility

### DISCUSSION THREAD

**Discussion by jubui** [awslabs/aws-mysql-jdbc#330] [General]
As far as I could tell the readme and the jdbc docs (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Connecting.html#Aurora.Connecting.JDBCDriverMySQL) for the aurora mysql driver don't say which aurora mysql database server versions it's compatible with. Is the aurora mysql jdbc driver v1 compatible with both v5 and v8 versions of the server? Will jdbc driver versions always be backward compatible with prior server versions?

**Comment by karenc-bq** [comment]
Hi @jubui, the AWS MySQL JDBC is based on the version 8.0.31 of the community MySQL Connector/J driver, and has the same server compatibility as the community driver. You can check when we update the driver with upstream changes in our changelog: https://github.com/awslabs/aws-mysql-jdbc/blob/main/CHANGELOG.md.

**Comment by hsuamz** [comment]
Closing this discussion as our README.md now contains this information.
